pages/activity/[id]/index.tsx

```diff
@@ -70,6 +76,9 @@ export const getServerSideProps = compose<
 export default class ActivityDetailPage extends PureComponent<
   InferGetServerSidePropsType<typeof getServerSideProps>
 > {
+  @observable
+  accessor chartKey: true | false = false;
```

Comment:
```suggestion
  accessor chartKey = false;
```
有默认值自动会推导出 `boolean` 类型。